### PR TITLE
doc(readme): items example

### PR DESCRIPTION
**Pull request by bas080**
adds an example to the README.md which shows how to use autocomplete with
a simple array.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,21 @@ jspm install aurelia-autocomplete
...
+
+```js
+export class Page {
+  languages = [ /* languages */ ];
+}
+```
+
+```html
+<auto-complete
+  items='languages'>
```
Comment (RWOverdijk) [inline README.md]: Double quote, and all same line

**Review by RWOverdijk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RWOverdijk) [reply README.md]: Also, shouldn't this be items.bind? It can change :)

Comment (bas080) [reply README.md]: Good catch, changes made.

File: README.md [hunk 2]
```diff
@@ -15,9 +15,20 @@ jspm install aurelia-autocomplete
 > simple
 
 ```html
-<auto-complete
-  resource='language'>
-</auto-complete>
+<auto-complete resource='language'></auto-complete>
```
Comment (RWOverdijk) [inline README.md]: double quotes

**Review by RWOverdijk — APPROVED** [review]

**Pull request merged**